test/integration/rules/frame-title/frame-title.html

```diff
@@ -32,3 +32,28 @@
 	role="presentation"
 ></iframe>
 <iframe src="frame-title/frames/level1a.html" id="pass7" role="none"></iframe>
+
+<iframe
+	src="frame-title/frames/level1a.html"
+	id="violation4"
+	role="none"
+	aria-live="assertive"
+></iframe>
```

Comment:
We may have to do more work for iframes here later. I don't think this is correct, but we'll get to it when Jey finishes his work on iframes in ACT. Acceptable for now.